recipe/meta.yaml

```diff
@@ -187,11 +190,12 @@ outputs:
     requirements:
       host:
         - {{ pin_subpackage("libgomp", exact=True) }}
-        - {{ pin_subpackage('openmp_impl', exact=True) }}
+        - {{ pin_subpackage('_openmp_mutex', exact=True) }}
       run:
         - _libgcc_mutex {{ _libgcc_mutex }}
+        - {{ pin_subpackage("_openmp_mutex", max_pin=None) }}
```

Comment:
libgomp in the host section should insert this in the run section for you, but being explicit is good too.

Reply:
Yeah, just wanted to be explicit as you said this was not added in the earlier PR.

Reply:
Ahhh I forgot about that! Good catch!